views/includes/start-update.html

```diff
@@ -9,13 +9,20 @@ <h2 class="govuk-heading-m">{{ i18n.startWhatYouCanDo1 }}</h2>
     <li>{{ i18n.startWhatYouCanDo4 }}</li>
     <li>{{ i18n.startWhatYouCanDo5 }}</li>
 </ul>
+{% if lang === 'en' %}
```

Comment:
make this the else so that it default to en if not cy?

Reply:
Fixed